### DISCUSSION TITLE
Drums in 5/4 time?

### DISCUSSION THREAD

**Discussion by fritterandwaste** [hydrogen-music/hydrogen#1187] [Q&A]
How do I adjust the settings to achieve a 5/4 time signature please? Many thanks.

**Comment by trebmuh** [comment]
> How do I adjust the settings to achieve a 5/4 time signature please? Many thanks.

It depends which Hydrogen version you're using. You need to tell us.

**Reply by fritterandwaste** [reply]
Sorry - using version 1.0.1 Sep 2020.

**Reply by oddtime** [reply]
In the pattern editor, set the size to "10", see:
http://hydrogen-music.org/documentation/manual/manual_en.html#sect.pattern_editor
The size display is in eighth notes that's why you have to enter 5/4 as 10/8.

In next version 1.1 (already in master branch if you want to build from github) you can set custom size with numerator / denominator

**Reply by fritterandwaste** [reply]
OK that's great, thank you very much. I think I had it in my head that that setting must be a power of 2 - i.e. 2,  4, 8, 16 etc but clearly not. Thanks again.
